lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb

```diff
@@ -267,31 +267,31 @@ def seer_to_cop_cooling_no_fan(seer)
   #
   # @param cop [Double] COP
   # @return [Double] Seasonal Energy Efficiency Ratio
-  def cop_to_seer_cooling_no_fan(cop)
+  def cop_no_fan_to_seer_cooling(cop)
     delta = 0.3796**2 - 4.0 * 0.0076 * cop
     seer = (-delta**0.5 + 0.3796) / (2.0 * 0.0076)
 
     return seer
   end
 
   # Convert from SEER to COP (with fan) for cooling coils
-  # per the method specified in 90.1-2013 Appendix G
+  # per the method specified in Thornton et al. 2011
   #
   # @param seer [Double] seasonal energy efficiency ratio (SEER)
   # @return [Double] Coefficient of Performance (COP)
-  def seer_to_cop_cooling_with_fan(seer)
+  def seer_to_cop_cooling(seer)
     eer = -0.0182 * seer * seer + 1.1088 * seer
-    cop = (eer / OpenStudio.convert(1.0, 'W', 'Btu/h').get + 0.12) / (1 - 0.12)
+    cop = eer_to_cop(eer)
```

Comment:
Straight EER to COP conversion, don't remove fan impact from metric since we don't to calculate a "cop no fan".